tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -133,37 +130,35 @@ We provide Capital Bikeshare sample data for you to easily import data and run s
 
     If your TiDB cluster is hosted by AWS (the Developer Tier is hosted by AWS by default), fill in the following parameters:
 
-    - **Data Source Type**: `AWS S3`.
-    - **Bucket URL**: enter the sample data URL `s3://tidbcloud-samples/data-ingestion/`.
-    - **Data Format**: select **TiDB Dumpling**.
-    - **Setup Credentials**: enter `arn:aws:iam::385595570414:role/import-sample-access` for Role-ARN.
-    - **Target Cluster**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: leave this field blank.
+    - **Data Format**: select **TiDB Dumpling**
+    - **Location**: `AWS`
+    - **Bucket URL**: enter the sample data URL `s3://tidbcloud-samples/data-ingestion/`
+    - **Role-ARN**: enter `arn:aws:iam::385595570414:role/import-sample-access`
+    - **Target Cluster**: shows the cluster name and the region name.
+
+    Confirm the compliance of cross region. Click **Next**.
 
     </div>
 
     <div label="GCP">
 
     If your TiDB cluster is hosted by GCP, fill in the following parameters:
 
-    - **Data Source Type**: `Google Cloud Stroage`.
-    - **Bucket URL**: enter the sample data URL `gcs://tidbcloud-samples-us-west1`.
     - **Data Format**: select **TiDB Dumpling**.
-    - **Target Cluster**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: leave this field blank.
+    - **Location**: `Google Cloud`
+    - **Bucket URL**: enter the sample data URL `gcs://tidbcloud-samples-us-west1`.
+    - **Target Cluster**: shows the cluster name and the region name.
+
+    Confirm the compliance of cross region. Click **Next**.
 
     </div>
     </SimpleTab>
 
-4. Click **Import**.
-
-    A warning message about the database resource consumption is displayed. For a newly created cluster, you can ignore the warning message.
-
-5. Click **Confirm**.
+4. Modify the file patterns and add the table filter rules if needed. For the sample data, you can skip this step. Click **Next**.
```

Comment:
The format of  sample data is TiDB Dumpling which is not supported to modified the  file pattern.

Reply:
Updated. But on the console, when you select TiDB Dumpling, the File Pattern area still appears, though users can not configure it.